src/components/MultiSuggestField/tests/MultiSuggestField.test.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+import "@testing-library/jest-dom";
+
+import { MultiSuggestField } from "../MultiSuggestField";
+import { Default, dropdownOnFocus, predefinedValues } from "../MultiSuggestField.stories";
+
+describe("MultiSuggestField", () => {
+    it("should render default input", () => {
+        const { container } = render(<MultiSuggestField {...Default.args} />);
+        const [input] = container.getElementsByClassName("eccgui-multiselect");
+
+        expect(input).toBeInTheDocument();
+    });
+
+    it("should render default selected items", async () => {
+        const { getByTestId } = render(<MultiSuggestField {...predefinedValues.args} />);
+
+        const [firstSelected, secondSelected]: Array<string> = predefinedValues.args.selectedItems.map(
+            ({ testLabel }) => testLabel.trim()
+        );
+
+        await waitFor(() => {
+            expect(getByTestId(firstSelected)).toBeInTheDocument();
+            expect(getByTestId(secondSelected)).toBeInTheDocument();
+        });
+    });
+
+    it("should clear all selected items on clear button click", async () => {
+        const { queryByTestId, getByTestId } = render(<MultiSuggestField {...predefinedValues.args} />);
+
+        const clearButton = getByTestId("clear-all-items");
```

Comment:
I guess you can get the button by using a selector, or configuring `getByTestId` to get used with `data-test-id`.